functions.js

```diff
@@ -0,0 +1,9 @@
+const strictEquals = (values) => {
+    if ('a', 'a') {
+        return true;
```

Comment:
Your linter should be complaining about the semi-colons. If it isn't you don't have it setup correctly. If it is, listen to it and remove them.